graviti/operation/frame.py

```diff
@@ -351,3 +385,33 @@ def do(  # pylint: disable=invalid-name, unused-argument
             sheet=sheet,
             record_keys=self.record_keys,
         )
+
+
+def _upload_files(
+    keys_and_paths: Iterable[Tuple[str, Path]],
```

Comment:
why we need `keys_and_paths`, can use just use File list instead?

Reply:
we need `keys_and_paths` to determine whether the data is remote or local. Because the data in operation has been changed to remote to get the True post data.

Reply:
class `File` means local file and `RemoteFile` means remote file. why not use the class to determine whether the data is remote of local?